web/src/index.css

```diff
@@ -102,7 +102,7 @@
 		--link-text-color: #4187eb;
 
 		--code-background-color: rgba(255, 255, 255, 0.1);
-		--media-placeholder-default-background: rgba(255, 255, 255, .1);
+		--media-placeholder-default-background: #333;
```

Comment:
The proper solution would be to apply `filter: blur(...)` to spoilered images that don't have a blurhash